recipes/lighttpd/lighttpd.inc

```diff
@@ -85,12 +85,23 @@ RECIPE_FLAGS += "lighttpd_sysvinit_start"
 DEFAULT_USE_lighttpd_sysvinit_start = "70"
 DEFAULT_USE_lighttpd_sysvinit_stop = "0"
 SYSVINIT_SCRIPT_lighttpd = "lighttpd"
-do_install[postfuncs] =+ " do_install_lighttpd_extra"
+do_install[postfuncs] =+ "${do_install_lighttpd_sysvinit_ENABLED}"
+do_install_lighttpd_sysvinit_ENABLED = ""
+do_install_lighttpd_sysvinit_ENABLED:USE_sysvinit = "do_install_lighttpd_sysvinit"
+do_install_lighttpd_sysvinit() {
+    install -d ${D}${sysconfdir}/init.d
+    install -m 0755 ${SRCDIR}/lighttpd ${D}${sysconfdir}/init.d/
+}
+
+inherit s6rc
+S6RC_LONGRUN_SERVICES = "lighttpd"
+SRC_URI:>USE_s6rc = " file://lighttpd.run"
+RECIPE_FLAGS += "lighttpd_s6rc_dependencies"
```

Comment:
No, I did not come up with any real dependencies. Specifically, I do not want it to depend on network in the general case.  And I don't think it use /dev, /sys, /proc or anything special.